.github/workflows/tests.yml

```diff
@@ -49,7 +49,7 @@ jobs:
         uses: actions/checkout@v2
         with:
           repository: CirclesUBI/circles-docker.git
-          ref: main
+          ref: test-transfers-update
```

Comment:
change this line back to main before merging

Reply:
Done :)